packages/xo-web/src/xo-app/vm/tab-advanced.js

```diff
@@ -510,6 +511,27 @@ export default class TabAdvanced extends Component {
   _onChangeCpuMask = cpuMask =>
     editVm(this.props.vm, { cpuMask: map(cpuMask, 'value') })
 
+  _onBootFirmwareChange = event => {
```

Comment:
To not duplicate the logic, better mutualize the `select` between the two interfaces.